ykman/cli/slot.py

```diff
@@ -54,7 +58,8 @@ def inner(ctx, param, val):
                                      callback=lambda c, p, v: int(v))
 
 
-def _failed_to_write_msg(ctx):
+def _failed_to_write_msg(ctx, exc_info):
+    logger.error('Failed to write message to driver', exc_info=exc_info)
```

Comment:
It didn't fail to write a message, the function name is referring to that it failed to write and should give the user a message. So I think the log error message should be something like "Failed to write to the device".

Reply:
Ah, good catch!